src/Symfony/Component/Config/Tests/Util/XmlUtilsTest.php

```diff
@@ -55,7 +55,7 @@ public function testLoadFile()
             XmlUtils::loadFile($fixtures.'valid.xml', array($mock, 'validate'));
             $this->fail();
         } catch (\InvalidArgumentException $e) {
-            $this->assertRegExp('/The XML file "[\w:\/\\\.]+" is not valid\./', $e->getMessage());
+            $this->assertRegExp('/The XML file "[\w:\/\\\.-]+" is not valid\./', $e->getMessage());
```

Comment:
Where is this change coming from?

Reply:
uh, that is actually unrelated to the feature, i can revert it (my tests were failing as i had a dash in my cwd pathname). Should i move it out?

Reply:
yes please, but we should take a look if we should make this change in the `2.7` branch instead (in a separate PR)